lkmpg.tex

```diff
@@ -3,7 +3,17 @@
 \usepackage[utf8]{inputenc}
 \usepackage[T1]{fontenc}
 \usepackage{graphicx}
+\usepackage{tocloft}
+
+% fancyhdr settings
 \usepackage{fancyhdr}
+\fancyhf{}
+\pagestyle{fancy}
+\lhead{\rightmark}
+\rhead{\thepage}
+\cfoot{The Linux Kernel Module Programming Guide}
```

Comment:
The string "The Linux Kernel Module Programming Guide" would appear in TeX script. Please revise.